src/Client.php

```diff
@@ -214,7 +214,7 @@ public function uniqueIds(): array
      */
     public function newUniqueId(): ?string
     {
-        return $this->usesUniqueIds ? (string) Str::orderedUuid() : null;
+        return $this->usesUniqueIds ? (string) Str::uuid7() : null;
```

Comment:
Do note that UUID7 leaks information that you may not want to be public, namely, the creation date.

In some cases that may be a privacy issue, or at the very least something that business does not want to be public (e.g., the date when a user last logged in / used the system etc?)

Reply:
Thanks for pointing that out, and client's creation date is not a security concern imo. However, this is customizable if someone think it is.